src/Regulator.h

```diff
@@ -128,12 +181,14 @@ class Regulator : public RingBuffer
 {
    public:
     /// construct a new regulator
-    Regulator(int rcvChannels, int bit_res, int FPP, int qLen, int bqLen,
+    Regulator(int rcvChannels, int bit_res, int localFPP, int qLen, int bqLen,
               int sample_rate);
 
     // virtual destructor
     virtual ~Regulator();
 
+    Time* mTime;
```

Comment:
This should probably move down to private variables section

Reply:
Also it should be initialized to NULL either in header or constructor, to avoid delete's of initialized memory (null is ok)